.controlplane/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update -qq && \
 
 # Install JavaScript dependencies
 # Make sure NODE_VERSION matches the node version in .nvmrc and package.json
-ARG NODE_VERSION=18.13.0
+ARG NODE_VERSION=18.20.3
```

Comment:
22?

Reply:
Hmmm Let's do that. I will upgrade to node v22 and see if there are any issues.